shadow-cljs.edn

```diff
@@ -13,11 +13,28 @@
                           "process" false
                           "buffer" false
                           "crypto" false}}
-   :modules {:flureedb {:exports {default flureedb/flureedb}}}
+   :modules {:fluree-browser-sdk {:exports {default fluree-browser-sdk/fluree-browser-sdk}}}
```

Comment:
Would it be worth eliminating the "single element namespace" warnings w/ this rename? I.e. use something like `fluree.browser-sdk` or even `fluree.sdk.browser` in CLJ(S) contexts?

Reply:
That sounds like a good opportunity to remove the `src-cljs` dir as well. I can do that in this pr.